reactive.js

```diff
@@ -0,0 +1,56 @@
+/* reactive engine. the code here combines functions exposed by the
+ * library into a single function to be run. i keep all code depending
+ * on Bacon here, so if somebody will want to stop using Bacon in the
+ * future, it will be easier */
+var lib = require('./lib2');
+var Bacon = require('bacon');
+
+// this is an adapter function. it gets a Pouch changes object and it
+// returns a Bacon stream
+function changesToStream(changes) {
+  return Bacon.fromBinder(function(sink) {
+    changes
+      .on('change', sink)
+      .on('error', function(text) {
+        sink(new Bacon.Error(text));
+      })
+      .on('complete', function() {
+        sink(new Backn.End());
```

Comment:
anyway, you do not need to get into Bacon now, i just wanted a feedback because i am pretty happy with the result, but somebody else might find it terrible ...